sharepoint/sharepoint-ps/sharepoint-online/Start-SPOCopilotAgentInsightsReport.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spocopilotagentinsightsreport
+applicable: SharePoint Online
+title: Start-SPOCopilotAgentInsightsReport
+schema: 2.0.0
+author: bhagatshweta
+ms.author: bhagatshweta
+ms.reviewer:
+manager: hikakar
+---
+ 
+# Start-SPOCopilotAgentInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables administrator to trigger the build of a new Copilot Agent Insight report for the last N days.
+
+## SYNTAX
+
+```powershell
+Start-SPOCopilotAgentInsightsReport [-ReportPeriodInDays <Int>] [-Force <SwitchParameter>]
+```
+
+## DESCRIPTION
+
+After this cmdlet is executed, the Copilot Agent Insight report generation request for the last N days gets queued in the pipeline and the below metadata is displayed with the following properties:
+
+| Property             | Description                                                 |
+|:---------------------|:------------------------------------------------------------|
+| Id                   | The unique Id of the report.                                |
+| CreatedDateTimeInUtc | The date and time the report creation was triggered in UTC. |
+| Status               | The status of the report.                                   |
+| ReportPeriodInDays   | The report duration in days.                                |
+
+## EXAMPLES
+
+### -----------------------EXAMPLE 1-----------------------------
+
+```powershell
+Start-SPOCopilotAgentInsightsReport
+```
+
+Example 1 generates the Copilot agent insights report for a default duration of 1 day as the parameter `–ReportPeriodInDays` is not provided.
+
+### -----------------------EXAMPLE 2-----------------------------
+
+```powershell
+Start-SPOCopilotAgentInsightsReport –ReportPeriodInDays 14
+```
+
+Example 2 generates the Copilot Agent insights report for a specified duration of 14 days.
+
+## PARAMETERS
+
+### -ReportPeriodInDays
+
+It is an optional parameter, and it specifies the duration of the Copilot Agent insights report in days. The possible values of ReportPeriodInDays are: 1, 7, 14, 28. If this parameter is not provided, it generates the report for a default duration of 1 day.
```

Comment:
>It is an optional parameter,  [](http://example.com/codeflow?start=0&length=29)

You don't need to specify this -- line 23 already insinuates it being optional

Reply:
other places too

Reply:
removed the optional keyword. Marking this as resolved, kindly re-review and let me know if I need to make any improvements.